engines/bastion_katello/app/assets/javascripts/bastion_katello/organizations/organization-selector.controller.js

```diff
@@ -13,7 +13,8 @@
 
         $scope.selectedOrganization = {};
 
-        Organization.queryUnpaged(function (response) {
+        // TODO: per_page hack necessary because of http://projects.theforeman.org/issues/21800
+        Organization.queryUnpaged({per_page: 99999999999}, function (response) {
```

Comment:
Apparently this needs to be `perPage` to pass eslint.. since that's not going to be possible (the API doesn't return that key), please disable the eslint rule in this line.

Reply:
`'per_page'` should work, I'll change this.